### PR TITLE
Migrate <NavLink /> 

### LINKED ISSUE #26: Migrate - <NavLink />

### PR DESCRIPTION
**Pull request by azamatsmith**
closes #26

### CODE REVIEW

File: src/tests/NavLink.test.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+    </NavLink>
+  );
+
+
+  it('should render without crashing', () => {
+    const div = document.createElement('div');
+    ReactDOM.render(<NavLink />, div);
+  });
+
+  it('should have a to prop', () => {
```
Comment (caseybaggz) [inline src/tests/NavLink.test.js:24]: Eep! We need to add those `const`

**Review by caseybaggz — CHANGES_REQUESTED** [review]
Just annoying tests updates. :P

**Comment by caseybaggz** [comment]
@azamatsmith I'm using react-router already on the demo side, so you can go ahead and remove it again so there is no conflict later. 👍

**Comment by azamatsmith** [comment]
@caseybaggz we can wait on this one. Right now looks like it will need to have `react-router` as a dependency in it's `package.json` file as it is trying to locate `react-router` from the non-root `node_modules` folder which obviously does not exist there.

**Pull request closed without merging**